libiocage/lib/Jail.py

```diff
@@ -601,7 +797,14 @@ def __getattr__(self, key):
 
         raise AttributeError(f"Jail property {key} not found")
 
-    def getattr_str(self, key):
+    def getstring(self, key):
```

Comment:
That would require to wrap all types returned to return a `-` in case the original value is `None`.